Packages/Feature/Sources/Feature/Extensions/CountrySelectingViewModelError+Extensions.swift

```diff
@@ -10,13 +10,26 @@ import UIComponents
 import ViewModels
 import DomainModels
 
-extension CountryListError {
-    var uiError: UIError {
+extension CountryListError: AlertPresentationTypeProviding {
+    public var alertPresentation: AlertPresentationType {
+        switch self {
+        case .userNotLoggedIn:
```

Comment:
I think we could do something like return AlertPresentationFactory(modality: toast(dialogState))